tests/garage/envs/wrappers/test_episodic_life.py

```diff
@@ -0,0 +1,32 @@
+import unittest
+
+import numpy as np
+
+from garage.envs.wrappers import EpisodicLife
+from tests.fixtures.envs.dummy import DummyDiscretePixelEnv
+
+
+class TestEpisodicLife(unittest.TestCase):
+    def test_episodic_life_reset(self):
+        env = EpisodicLife(DummyDiscretePixelEnv())
+        obs = env.reset()
+
+        # env has reset
+        assert np.array_equal(obs, np.zeros(env.observation_space.shape))
+        assert env.unwrapped.ale.lives() == 5
+
+        obs, _, d, info = env.step(0)
+        assert d
```

Comment:
why done is `True` here?

Reply:
DummyDiscretePixelEnv is designed to lose 1 life after each step. Since a life is lost, done will be True.
Also, in the wrapper design, calling env.reset() when number of lives is not 0 will result in one single step(). That's why in this test step() is only called 4 times but 5 lives are lost because 1 reset() is called.

I will add more docstring about why DummyDiscretePixelEnv is designed in this way for the ease of testing.